packages/react-router/typedoc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "entryPoints": ["./index.ts"],
+  "entryPoints": ["./index.ts", "./dom-export.ts"],
```

Comment:
This doesn't do quite what I would have hoped :/.  It adds another level of nesting under `react-router`:

![Screenshot 2025-04-30 at 2 08 03 PM](https://github.com/user-attachments/assets/685fe3ca-a5bb-4126-982c-bbcc89673d4a)


Unsure if theres any way to flatten that to:

```
react-router
  ...All the current stuff
react-router/dom
  ...New Stuff
```